.github/workflows/ci-cd.yml

```
@@ -374,6 +374,80 @@ jobs:
       if: ${{ matrix.os == 'ubuntu' }}
       uses: aio-libs/prepare-coverage@v24.9.2
 
+
+  test_debug_mode:
+    name: Test in debug mode
+    strategy:
+      matrix:
+        pyver:
+        - 3.13t
+        - 3.13
+        - 3.12
+        - 3.11
+        - "3.10"
+        - 3.9
+      fail-fast: false
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+
+    steps:
+    - name: Checkout project
+      uses: actions/checkout@v4
+    - name: Setup Python ${{ matrix.pyver }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.pyver }}
+        allow-prereleases: true
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
+        echo "dir=$(pip cache dir)" >> "${GITHUB_OUTPUT}"
+      shell: bash
+    - name: Cache PyPI
+      uses: actions/cache@v4
+      with:
+        key: >-
+          pip-ci-${{
+            runner.os
+          }}-${{
+            matrix.pyver
+          }}--${{
+            hashFiles('requirements/*.txt')
+          }}
+        path: ${{ steps.pip-cache.outputs.dir }}
+        restore-keys: >-
+          pip-ci-${{
+            runner.os
+          }}-${{
+            matrix.pyver
+          }}--
+    - name: Install dependencies
+      uses: py-actions/py-dependency-install@v4
+      with:
+        path: requirements/pytest.txt
+    - name: >-
+        Self-install source
```

Actually, source is never installed. Only the contents of wheels are. I think, the following will better communicate what this does.

```suggestion
    - name: Compile and install from source checkout
```